src/views/Simulation/Recapitulatif.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <form @submit.prevent="onSubmit" class="recapitulatif-form">
+    <div>
+      <template v-for="(chapter, chapterIndex) in chapters">
+        <div
+          class="chapter-block"
+          :key="chapter.name"
+          v-if="chapter.questions.length"
+        >
+          <h2 class="aj-question">{{ chapter.label }}</h2>
+          <template v-for="(question, questionIndex) in chapter.questions">
+            <template v-if="question.isChapterSubtitle">
+              <div
+                :key="`chapter_${chapterIndex}_question_${questionIndex}`"
+                class="recapitulatif-row"
+              >
+                <div v-html="question.label" class="subtitle"></div>
+              </div>
+            </template>
+            <template v-else>
+              <div
+                :key="`chapter_${chapterIndex}_question_${questionIndex}`"
+                class="recapitulatif-row"
+              >
+                <div class="question-col" v-html="question.label"></div>
+
+                <div
+                  class="value-col"
+                  v-if="typeof question.value !== 'object'"
+                  >{{ question.value }}</div
+                >
+                <div class="edit-col"
+                  ><router-link :to="question.path">Modifier</router-link></div
+                >
+              </div>
+              <template v-if="typeof question.value === 'object'">
+                <div
+                  :key="`chapter_${chapterIndex}_question_${questionIndex}_obj`"
+                  class="recapitulatif-row recapitulatif-row-wrap"
+                >
+                  <div
+                    class="value-cell"
+                    v-for="(value, name) in question.value"
+                    :key="name"
+                  >
+                    <div style="font-style: italic">{{ name }} :</div>
+                    <div>{{ value }}</div>
+                  </div>
+                </div>
+              </template>
+            </template>
+          </template>
+        </div>
+      </template>
+    </div>
+
+    <Actions v-bind:onSubmit="onSubmit" />
```

Comment:
previous does not really have a meaning here... Can we have a Confirmer button instead ?

Reply:
Qu'est ce qui fait te dire que tu ne vois pas de sens dans le bouton précédent ? :)
Personnellement j'aime bien le bouton précédent et suivant. Cela pernet de garder logique du simulateur et donne envie de continuer.
Après mon avis n'est pas tranché là-dessus. Dans le cas oú l'on conserverait ton idée, que penses-tu du texte "Obtenir le résultat de ma simulation" comme ci-dessous.

![image](https://user-images.githubusercontent.com/65901733/133770243-44fb4f62-e689-4723-9098-bf489d29b07d.png)
Autre proposition 
![image](https://user-images.githubusercontent.com/65901733/133771050-e799c96e-ea91-44c9-bc3b-fc878f2ffd65.png)

D'un autre coté ça permet de mieux séparer une simple étape du récapitulatif.

Reply:
Autre texte pour le bouton : "Découvrir mes aides"